wpeverest/everest-forms
# Enhance - Option to load fonts locally

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds an option in Misc settings to load fonts locally. This setting will be used when loading fonts in **Everest Forms Email Templates, Everest Forms Conversational Forms and Everest Forms Style Customizer**.

### How to test the changes in this Pull Request:

1. Toggle the newly added **Load Fonts Locally** in _Misc_ settings and verify whether the fonts are being loaded from remote google url or local url in the above mentioned addons features.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Option to load fonts locally
